File: iniciante/javascript/1149.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

let a = parseInt(lines.shift())
let b = parseInt(lines.shift())
let soma = 0

while(b <= 0){
    b = parseInt(lines.shift())
}
for(let i = 0 ; i <= b - 1; i++){
    soma += a
    a++
}

console.log(soma)

File: iniciante/python/1149.py
# -*- coding: utf-8 -*-

entrada = input().split(' ')
entrada = [int(x) for x in entrada]
a = entrada[0]
b = 0
soma = 0
for x in entrada[1:]:
    if x > 0:
        b = x
        break

for x in range(entrada[0], 0, -1):
    soma += x
for x in range(0, b):
    soma += x
print(soma)
